packages/manager-react-components/src/hooks/data-api/ports/iceberg/useIceberg.type.ts

```diff
@@ -0,0 +1,22 @@
+import { Dispatch, SetStateAction } from 'react';
+import {
+  IcebergFetchResult,
+  IcebergFetchParams,
+} from '@ovh-ux/manager-core-api';
+import { DatagridColumn, ColumnSort } from '../../../../components';
+import { InfiniteData } from '../../adapters/use-infinite-query';
```

Comment:
In a pure hexagonal architecture, ports should never be coupled with adapters as it's the contract between domain and infra layers, that is fully agnostic from the implementation.

Port should describe its needs, adapter implements it :)